content/departments/product/design/index.md

```diff
@@ -65,6 +65,7 @@ Product design at Sourcegraph is low-process and highly adaptive. Instead of des
 - [Evaluative research](./research/conducting-evaluative-research.md)
 - [Design and interaction guidelines](design-and-interaction-guidelines.md)
 - [Design collaboration and review](./design_collaboration_review/index.md)
+- [Collaboration with engineering](./design-and-engineering-collaboration.md)
```

Comment:
Would it make sense to also link this in the "Processes" section of https://handbook.sourcegraph.com/departments/engineering/ so we have links from both sides?

Reply:
Addressed [here](https://github.com/sourcegraph/handbook/pull/8215/files#diff-b2a725171075afa41533266a42e359a43fe13e858a7fe87e630aff12f6ec9c87R42). Thanks again!